pages/product/index.vue

```diff
@@ -235,13 +236,22 @@ import { defineComponent, computed } from "@vue/composition-api"
 
 import LazyHydrate from "vue-lazy-hydration"
 
-import { useAsync } from "@nuxtjs/composition-api"
+import { ref, useAsync } from "@nuxtjs/composition-api"
 import KiboFulfillmentOptions from "@/components/KiboFulfillmentOptions.vue"
 import KiboProductActions from "@/components/KiboProductActions.vue"
-import { useProductSSR, useUiState, usePurchaseLocation, productGetters } from "@/composables"
+import {
+  useProductSSR,
+  useUiState,
+  usePurchaseLocation,
+  productGetters,
+  useCart,
+} from "@/composables"
 import { isFulfillmentOptionValid } from "@/composables/helpers"
+import { isProductVariationsSelected } from "~~/composables/helpers/validateProductVariations"
+import { AddItemsToCartParams } from "~~/composables/types/useCart"
```

Comment:
Be consistent with the alias. Use @ instead of ~~

Reply:
Done